components/director/internal/repo/sql_types.go

```diff
@@ -0,0 +1,46 @@
+package repo
+
+import "database/sql"
+
+func NewSqlNullString(text *string) sql.NullString {
```

Comment:
to consider: NewOptionalString, FromOptionalString or NewNullableString, FromNullableString

Reply:
`NewNullableString` looks better